rosflight/src/mavrosflight/mavrosflight.cpp

```diff
@@ -36,26 +36,45 @@
 
 #include <rosflight/mavrosflight/mavrosflight.h>
 
+#if defined(USE_ROS)
+#include <rosflight/ros_logger.h>
+#elif defined(STANDALONE)
+#include <rosflight/mavrosflight/logger_interface.h>
+#endif
```

Comment:
As a possible alternative, could we have some type of adapter header, like `rosflight/mavrosflight/logger_adapter.h` that looks something like

```c++
namespace mavrosflight
{
#if defined(USE_ROS)
#include <rosflight/ros_logger.h>
using DerivedLoggerType = rosflight::ROSLogger;

#elif defined(STANDALONE)
#include <rosflight/mavrosflight/logger_interface.h> // or #include <rosflight/mavrosflight/default_logger.h> if it's in its own header file
using DerivedLoggerType = mavrosflight::DefaultLogger;

#else
#error Unknown logging backend supplied for mavrosflight. Define USE_ROS or STANDALONE.
#endif
} // namespace mavrosflight
```

Then each of `mavrosflight.cpp`, `param_manager.cpp`, and `time_manager.cpp` just include that header, which eliminates the duplicated code at the top, and then the blocks at the bottom all just simplify to 
```c++
template class MavROSflight<mavrosflight::DerivedLoggerType>;
```
etc.

Reply:
Great idea. Just a note, I ran into weird namespacing issues using a `#include` inside of a namespace.